src/Fl_x.cxx

```diff
@@ -38,6 +38,7 @@
 #  include <FL/filename.H>
 #  include <stdio.h>
 #  include <stdlib.h>
+#  include <cmath>
```

Comment:
Please use "math.h" instead. Sorry for "nitpicking", but that's **old** FLTK style.